code/datums/status_effects/debuffs.dm

```diff
@@ -57,7 +65,7 @@
 
 /datum/status_effect/incapacitating/sleeping/tick()
 	if(owner.staminaloss)
-		owner.adjustStaminaLoss(-0.35) //reduce stamina loss by 0.35 per tick, 7 per 2 seconds
```

Comment:
Stating that an adjuststaminaloss in a Tick() proc reduces staminaloss every tick does not convey extra information.